### PR TITLE
Update postinstall notes

### PR DESCRIPTION
**Pull request by elingg**
@jsancio or @ConnorDoyle, please take a look when you have a chance. Thanks!

### CODE REVIEW

**Comment by jsancio** [comment]
cc @air

File: repo/packages/C/chronos/0/package.json [hunk 1]
```diff
@@ -10,5 +10,6 @@
     "icon-small": "https://downloads.mesosphere.io/chronos/assets/icon-service-chronos-small.png",
     "icon-medium": "https://downloads.mesosphere.io/chronos/assets/icon-service-chronos-medium.png",
     "icon-large": "https://downloads.mesosphere.io/chronos/assets/icon-service-chronos-large.png"
-  }
+  },
+   "postInstallNotes": "The Chronos (Job Scheduling/Cron Scheduling) DCOS Service implementation has been successfully installed!\nIn order for Chronos to start successfully, all resources must be available in the cluster, including ports, CPU shares, Ram and Disk.\nWe recommend a minimum of one slave with at least 1 CPU and 2GB of RAM available for use by the Chronos Service.\n\n\tDocumentation: https://github.com/mesos/chronos\n\tIssues: https:/github.com/mesos/chronos/issues"
```
Comment (air) [inline repo/packages/C/chronos/0/package.json]: Let's simplify to `Chronos DCOS Service has been successfully installed!\nWe recommend a minimum of one node with at least 1 CPU and 2GB of RAM available for the Chronos Service.\n\n\tDocumentation: https://github.com/mesos/chronos\n\tIssues: https:/github.com/mesos/chronos/issues`

The middle line about starting successfully was aimed at Cassandra, where startup takes a long time.

File: repo/packages/H/hdfs/0/package.json [hunk 2]
```diff
@@ -10,5 +10,6 @@
     "icon-small": "https://downloads.mesosphere.io/hdfs/assets/icon-service-hdfs-small.png",
     "icon-medium": "https://downloads.mesosphere.io/hdfs/assets/icon-service-hdfs-medium.png",
     "icon-large": "https://downloads.mesosphere.io/hdfs/assets/icon-service-hdfs-large.png"
-  }
+  },
+  "postInstallNotes": "The Hadoop Distributed File System (HDFS) DCOS Service implementation has been successfully installed! Note that the service is alpha and there may be bugs, incomplete features, incorrect documentation or other discrepancies.\nIn order for HDFS to start successfully, all resources must be available in the cluster, including ports, CPU shares, RAM and disk.\nWe recommend a minimum of five slaves with at least 2 CPU's and 8GB of RAM available for use by the HDFS Service.\n\n\tDocumentation: https://github.com/mesosphere/hdfs\n\tIssues: https://github.com/mesosphere/hdfs/issues"
```
Comment (air) [inline repo/packages/H/hdfs/0/package.json]: Let's simplify a little to
`HDFS DCOS Service has been successfully installed!\nNote that the service is alpha and there may be bugs, incomplete features, incorrect documentation or other discrepancies.\nIn order for HDFS to start successfully, all resources must be available in the cluster, including ports, CPU shares, RAM and disk.\nWe recommend a minimum of five nodes, each with at least 2 CPU shares and 8GB of RAM available for use by the HDFS Service.\n\n\tDocumentation: https://github.com/mesosphere/hdfs\n\tIssues: https://github.com/mesosphere/hdfs/issues`

**Comment by elingg** [comment]
thanks @air! Updated the notes as you specified. Could you take a final look?

**Comment by air** [comment]
:+1:

**Pull request merged**